test/benchmark/config/gatewayclass.yaml

```diff
@@ -29,15 +29,6 @@ spec:
               memory: "256Mi"
               cpu: "500m"
   telemetry:
-    metrics:
-      prometheus: {}
-      sinks:
-        - type: OpenTelemetry
-          openTelemetry:
-            backendRefs:
-              - name: otel-collector
```

Comment:
`otel-collector` is not installed in the bechmark test. Removing it form the EnvoyProxy, otherwise the gateway will not be accepted and the test will fial.